src/implementations/twig/components/content-block/content-block.html.twig

```diff
@@ -0,0 +1,167 @@
+    - "labels" (array) (default: []): Array of ECL Labels
+    - "primary_meta" (array of strings) (default: []): Primary meta of the content block
+    - "title" (associative array) (default: {}): Title of the content block, following ECL Link structure
+    - "description" (string) (default: ''): Description of the content block
+    - "secondary_meta" (array) (default: []): format: [
+        {
+          "label" (string) (default: ''): Label of secondary meta item
+          "icon" (array) (default: {}) Icon of the secondary meta, following ECL Icon structure
+        },
+        ...
+      ]
+    - "tags" (array) (default: []): Array of ECL Tags
+    - "links" (array) (default: []): Array or multi array of ECL Links
+    - "lists" (array) (default: []): Array of ECL Description list
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (optional) (string)
+        ...
+      ],
+#}
+
+{# Internal properties #}
+
+{% set _labels = labels|default([]) %}
+{% set _primary_meta = primary_meta|default([]) %}
+{% set _title = title|default({}) %}
+{% set _description = description|default('') %}
+{% set _secondary_meta = secondary_meta|default([]) %}
+{% set _tags = tags|default([]) %}
+{% set _links = links|default([]) %}
+{% set _lists = lists|default([]) %}
+{% set _css_class = 'ecl-content-block' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% if attr.value is defined %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name|e('html_attr') ~ '="' ~ attr.value|e('html_attr') ~ '"' %}
+    {% else %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name|e('html_attr') %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<div class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  {% if _labels is defined and _labels is not empty and _labels is iterable %}
+    <ul class="ecl-content-block__label-container">
+      {% for _label in _labels %}
+        <li class="ecl-content-block__label-item">
+          {% include '@ecl/label/label.html.twig' with _label only %}
+        </li>
+      {% endfor %}
+    </ul>
+  {% endif %}
+
+  {% if _primary_meta is not empty and _primary_meta is iterable %}
+    <ul class="ecl-content-block__primary-meta-container">
+      {% for _meta in _primary_meta %}
+        <li class="ecl-content-block__primary-meta-item">
+          {{- _meta -}}
+        </li>
+      {% endfor %}
+    </ul>
+  {% endif %}
+
+  {% if _title is not empty and _title.label is not empty %}
+    <h1 class="ecl-content-block__title">
```

Comment:
I see that's already the case for the content-item, but are we sure we want an <h1> here, at the limit an h2, right?